.eslintrc.js

```diff
@@ -1,24 +1,18 @@
 module.exports = {
   root: true,
-  parser: 'babel-eslint',
-  parserOptions: {
-    sourceType: 'module'
-  },
-  extends: 'vue',
-  // add your custom rules here
-  'rules': {
-    // allow async-await
-    'generator-star-spacing': 0,
-    // allow debugger during development
-    'no-debugger': process.env.NODE_ENV === 'production' ? 2 : 0
-  },
+  extends: [
+    'eslint:recommended',
+    'plugin:vue/recommended',
+    'prettier',
+  ],
   globals: {
     requestAnimationFrame: true,
     performance: true,
-    window: true
+    window: true,
   },
-  "env": {
-    "browser": true,
-    "node": true
+  env: {
+    browser: true,
+    commonjs: true,
+    es6: true,
   },
 }
```

Comment:
Open to suggestions on this configuration ☝️ 

`.eslintrc.json` would never have been relevant as `.eslintrc.js` takes precedence per [Configuration File Formats](https://eslint.org/docs/user-guide/configuring#configuration-file-formats); thus, I just merged the former into here.

I'm not sure it makes sense to extend `plugin:vue/recommended`, for example, given that this project doesn't really have any `.vue` files, but I left it in here for closer compatibility.

Reply:
Good for eslint files

For config related to vue, let's keep it for now ;)